test/selectors-relative.spec.js

```diff
@@ -1,33 +1,38 @@
-const Promise = require("promise");
-const express = require("express");
-const { firefox } = require("playwright");
+const fs = require("fs");
+const process = require("process");
+const { chromium } = require("playwright");
 const relativeSelector = require("../src/index");
-let server, port;
+
+const headless =
+  process.env.HEADFULL === undefined ||
+  process.env.HEADFULL.indexOf("true") < 0;
+
 let browser;
+let ctx;
 let page;
 
 beforeAll(async () => {
-  browser = await firefox.launch();
-  page = await browser.newPage();
-
-  const app = express();
-  app.use(express.static("test/assets"));
-  await new Promise((resolve) => {
-    server = app.listen(0, () => {
-      port = server.address().port;
-      console.log(`Running server on ${port}...`);
-      resolve();
+  jest.setTimeout(10000);
```

Comment:
Is it needed ? 10s is long... I found that Jest async should enough to wait for the 'server'

Reply:
It's to wait that my browser opening in HeadFull mode 😆
My personal PC is really old.
We can have different values between headless / headfull mode but not sure that is really userfull.
We really need to add more integration test with realistic html page.
Then using headfull mode will help to develop it.

Other points is about Jest .. on watch mode I don't find any tips to keep the browser open where with Mocha I do it successfully.
Watch mode is really interesting to re play fast a test under development.

Reply:
No worry, first look tests that I have added.
Then I think I should discut about that: 
```
if (
    proximitySelector === "toLeftOf" ||
    proximitySelector === "toRightOf"
  ) {
    yDiff = yDiff * 10; // Give priority on distance X
  }
```
It to give a priority on the requested direction. When you find a composant at Left you expect that the components must be really at Left.
If we have a component at 20px left and top 0.
And a second at 2px left but 15px top.
That trick force to select the first one.

The value 10 look good but should be precise with real use case.